flink/v2.0/flink/src/test/java/org/apache/iceberg/flink/maintenance/api/TestZkLockFactory.java

```diff
@@ -39,15 +47,31 @@ void before() {
     super.before();
   }
 
-  @Override
-  TriggerLockFactory lockFactory() {
-    return new ZkLockFactory(zkTestServer.getConnectString(), testLockId, 5000, 3000, 1000, 3);
+  @Test
+  void testMultiTableLock() throws IOException {
```

Comment:
Maybe we can move this test to the Base class, and run for the JDBCLockFactory too?

Reply:
Your suggestion is reasonable, and I will optimize the code accordingly.